pallets/anchor/src/lib.rs

```diff
@@ -220,8 +217,12 @@ pub mod pallet {
 }
 
 impl<T: Config<I>, I: 'static> AnchorInterface<T, I> for Pallet<T, I> {
-	fn create(creator: T::AccountId, depth: u8) -> Result<T::TreeId, DispatchError> {
-		T::Mixer::create(T::AccountId::default(), depth)
+	fn create(creator: T::AccountId, depth: u8, max_edges: u32) -> Result<T::TreeId, DispatchError> {
+		// FIXME: is that even correct?
```

Comment:
We will want to integrate some token pallet to begin handling deposit sizes and probably add it as a function argument. What do you think?

Reply:
Yes, I agree, we need to handle this correctly.

Reply:
We could do that in another PR.